functional/otlp/func_http_main.cc

```diff
@@ -170,6 +170,8 @@ void payload()
   auto span = tracer->StartSpan(k_span_name);
   span->SetAttribute(k_attr_test_name, opt_test_name);
   span->End();
+
+  tracer->ForceFlushWithMicroseconds(1000000);
```

Comment:
`Tracer::ForceFlushWithMicroseconds` do nothing and it do not wait data exporting in async mode now.
Maybe we need another way to waiting in async mode.